.travis.yml

```diff
@@ -1,12 +1,9 @@
 language: python
 
-python: 3.5
+python: 3.7
 env:
-    - TOXENV=py27
-    - TOXENV=py33
-    - TOXENV=py34
-    - TOXENV=py35
-    - TOXENV=pypy
+    - TOXENV=py37
+    - TOXENV=py38
     - TOXENV=pypy3
```

Comment:
Could you please update above code to be similar to https://github.com/morepath/morepath_cerebral_todomvc/commit/c5b413728e05e92423c162534c32e6b39ee1cb0f - looks like Travis does no longer install several python versions on its boxes, and so we need to specify the needed versions in a `matrix`.

Reply:
Done